input/lightcurves/Get_FITS_from_DESY-LC-Archive.py

```diff
@@ -0,0 +1,14 @@
+import sys
+import DESYLCArchiveToECSV
+
+## This is a Python-script to dump the FITS Lightcurves from the DESY LC Archive to ECSV files
+## URL: https://astro.desy.de/gamma_astronomy/magic/projects/light_curve_archive/index_eng.html
+## This Script needs DESYLCArchiveToECSV.py
+## links and source names are stored in a dictionary
+## links can be added if the DESY LC Archive gets extended
+## source names can be easily edited and changed to tev-id for example
+
+url = {"1es1959+650":"http://www-zeuthen.desy.de/multi-messenger/GammaRayData/1es1959+650_combined_lc_v0.2.fits", "mrk421":"http://www-zeuthen.desy.de/multi-messenger/GammaRayData/mrk421_combined_lc_v0.2.fits", "mrk501":"http://www-zeuthen.desy.de/multi-messenger/GammaRayData/mrk501_combined_lc_v0.2.fits"}
```

Comment:
Lines this long are not well readable.
I'm reviewing your PR now, and to read this line I have to scroll left and right, which is very annoying.

I suggest you format like this:
```python
url = {
    "1es1959+650":"http://www-zeuthen.desy.de/multi-messenger/GammaRayData/1es1959+650_combined_lc_v0.2.fits",
    # other lines
}
```